config/v1/types_oauth.go

```diff
@@ -89,7 +89,7 @@ type OAuthTemplates struct {
 	// If unspecified, the default login page is used.
 	// The namespace for this secret is openshift-config.
 	// +optional
-	Login SecretNameReference `json:"login,omitempty"`
```

Comment:
@adambkaplan these changes here in `types_oauth.go` do not change serialization behaviour. But please double check you really meant what you specified with tags (and fix in a follow-up if you haven't).

Reply:
I'd check with @openshift/sig-auth on this - DevEx doesn't own the type. I assume I'm on the blame when I refactored how secrets and configMaps are referenced in the config API.

Reply:
There was something about kubebuilder not respecting "+optional" and needing the "omitempty".
@twiest do you still require the `omitempty` for optional types?